simplei.module

```diff
@@ -37,7 +37,8 @@ function simplei_init() {
 
     // Change the admin bar's style.
     if ($admin_bar_access && !empty($indicator[0])) {
-      $css = "#admin-bar { background: $indicator[0]; }";
+      $css = ":root { --simplei-background-color: $indicator[0]; }";
```

Comment:
```suggestion
      $css = "#admin-bar { --simplei-background-color: $indicator[0]; }";
```

I think we can make it specific to just the `#admin-bar` so it doesn't need to be at the `:root`. @laryn